apps/marketplace/components/BuyerDashboard/BuyerDashboardHeader.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { rootPath } from 'marketplace/routes'
+import styles from './BuyerDashboard.scss'
+
+export class BuyerDashboardHeader extends Component {
+  constructor(props) {
+    super(props)
+    this.handleMenuClick = this.handleMenuClick.bind(this)
+  }
+
+  handleMenuClick(e) {
+    e.preventDefault()
+    const targetEl = e.currentTarget
+    this.props.history.push(targetEl.getAttribute('href'))
```

Comment:
shouldn't the router to do this?

Reply:
It looks like BrowserRouter and Link might do this - I'll look into those.